peps/pep-0757.rst

```diff
@@ -238,6 +238,17 @@ Implementation
 Benchmarks
 ==========
 
+Code::
+
+    /* Query parameters of Python’s internal representation of integers. */
+    const PyLongLayout *layout = PyLong_GetNativeLayout();
+
+    size_t int_digit_size = layout->digit_size;
+    size_t int_digits_order = layout->digits_order;
```

Comment:
This member is signed, but size_t is unsigned. Is the cast done on purpose?

Reply:
No, it's wrong.